SPTDataLoader/SPTDataLoaderResponse.m

```diff
@@ -181,4 +181,10 @@ - (NSDate *)retryAfterForHeaders:(NSDictionary *)headers
     return [httpDateFormatter dateFromString:retryAfterValue];
 }
 
+- (NSString *)debugDescription
+{
+    return [NSString stringWithFormat:@"%@ { URL: %@ } { status code: %lu, headers: %@ }", [super debugDescription], self.response.URL, (long)self.statusCode, self.responseHeaders];
```

Comment:
You've specified that the status code should be `long unsigned` but you then cast it to only `long`.